Cargo.toml

```diff
@@ -53,7 +53,7 @@ nu-table = { path = "./crates/nu-table", version = "0.65.1"  }
 nu-term-grid = { path = "./crates/nu-term-grid", version = "0.65.1"  }
 nu-utils = { path = "./crates/nu-utils", version = "0.65.1"  }
 reedline = { git = "https://github.com/nushell/reedline.git", features = ["bashisms", "sqlite"]}
-pretty_env_logger = "0.4.0"
+pretty_env_logger = { git = "https://github.com/seanmonstar/pretty-env-logger", rev = "f9e35b6dbbf06de55222c944c9e1e176ce73b3a7" }
```

Comment:
I'm not positive but I don't think we can have a git dependency here because it doesn't allow us to publish to crates.io.

Reply:
The author doesn’t make a new release and publish to crates.io yet.

Reply:
Maybe you can ask them if they plan on making a release in the next 10 days. We have a release in 11 days.

Reply:
Any word from the pretty_env_logger people? I kind of like this idea of logging, but we just can't have git dependencies.

Reply:
No news. Seems they are busy. :joy:

Reply:
since there hasn't been a published crate from them in more than 2 years, perhaps you could find something other than pretty-env-logger to use?